miio/gateway/alarm.py

```diff
@@ -61,3 +65,28 @@ def set_triggering_volume(self, volume):
     def last_status_change_time(self) -> datetime:
         """Return the last time the alarm changed status."""
         return datetime.fromtimestamp(self._gateway.send("get_arming_time").pop())
+
+    def subscribe_events(self):
+        """subscribe to the alarm events using the push server."""
+        if self._gateway._push_server is None:
+            _LOGGER.error("Can not install push callback withouth a push_server")
+            return False
+
+        event_info = EventInfo(
+            action="alarm_triggering",
+            extra="[1,19,1,111,[0,1],2,0]",
+            trigger_token=self._gateway.token,
+        )
+
+        event_id = self._gateway._push_server.subscribe_event(self._gateway, event_info)
+        if event_id is None:
+            return False
```

Comment:
Likewise here, would an exception be better?

Reply:
In this case I would like to stick to the boolean False since that is also done this way for subdevices since the rest of the subscriptions can then continue if only one fails...